rasa/nlu/extractors/extractor.py

```diff
@@ -254,6 +268,40 @@ def _update_confidence_values(
                 confidences[ENTITY_ATTRIBUTE_GROUP][idx],
             )
 
+    @staticmethod
+    def _check_is_single_entity(
+        text: Text,
+        token: Token,
+        last_token_end: int,
+        split_entities_config: Dict[Text, bool],
+        current_entity_tag: Text,
+    ):
+        # Tokens need to be no further than 3 positions apart
+        tokens_within_range = token.start - last_token_end <= 3
+
+        # The interleaving tokens *must* be a full stop, a comma, or a whitespace
+        interleaving_text = text[last_token_end : token.start]
+        tokens_separated_by_allowed_chars = all(
+            filter(
+                lambda char: True if char in {".", ",", " "} else False,
+                interleaving_text,
+            )
+        )
+
+        # The current entity type must match with the config (default value is True)
+        default_value = split_entities_config[SPLIT_ENTITIES_BY_COMMA]
+        split_current_entity_type = split_entities_config.get(
+            current_entity_tag, default_value
+        )
+
+        # matching_entity_types = True # TODO: Check the current entity tag matches the pre-configured one form current_entitiy_tag and the split_entities_config dict
```

Comment:
This seems to be relevant and should be added, shouldn't it?

Reply:
oh dear, forgot to delete this...its superseded by what is above.